demo.scenario

```diff
@@ -132,14 +132,14 @@ debian-redis         latest    8366943c77e8    3 days ago      136MB
 alpine-redis         latest    aae644cd3417    3 days ago      6.99MB
 nix-redis            latest    30486183a209    50 years ago    45.4MB
 nix-redis-minimal    latest    a21238890680    50 years ago    2.02MB
-$ # Did we just produced a docker image _smaller_ then Alpine? Interesting!
+$ # Did we just produce a docker image _smaller_ then Alpine? Interesting!
```

Comment:
```suggestion
$ # Did we just produce a docker image _smaller_ than Alpine? Interesting!
```

Reply:
hehe, I always get this wrong :) thank you!